src/_data/side-nav.yml

```diff
@@ -309,8 +309,6 @@
           permalink: /tools/webdev
   - title: Managing source code
     children:
-      - title: Formatting code
-        permalink: /guides/language/formatting
       - title: What not to commit
         permalink: /guides/libraries/private-files
```

Comment:
Since this is the only child page of `Managing source code` now, what do you think of getting rid of that category and moving `What not to commit` up a level in the side nav? Might look kind of clunky/random next to the other sections up "Tools and techniques", but I think it's better than the wasted section for now